packages/components/psammead-content-anchor/src/index.stories.jsx

```diff
@@ -90,7 +91,9 @@ storiesOf('Components/ContentAnchor', module)
     ({ text, script, service }) => {
       return (
         <Wrapper>
-          <Headline>{text}</Headline>
+          <Headline script={latin} service="news">
```

Comment:
`script` and `service` are available as parameters in this function 😄 
```suggestion
          <Headline script={script} service={service}>
```